cmd/ceremony/key.go

```diff
@@ -54,9 +54,14 @@ type keyInfo struct {
 }
 
 func generateKey(ctx pkcs11helpers.PKCtx, session pkcs11.SessionHandle, label string, outputPath string, config keyGenConfig) (*keyInfo, error) {
+	var err error
```

Comment:
nit: could be collapsed onto next line, now that its first reference isn't inside the switch